monitor/src/main/webapp/resources/monitor/reload.js

```diff
@@ -0,0 +1,22 @@
+window.setInterval(function() {
+	$.ajax({
+		type : "GET",
+		dataType : "json",
+		url : "/monitor/admin/apps/reload",
```

Comment:
this is not how you create links within the application. This doesn't work as soon as the app is not called 'monitor' (which it is not for example on the review instance).